big-practice/src/js/script.js

```diff
@@ -0,0 +1,33 @@
+const emailInput = document.getElementById('email');
+const passwordInput = document.getElementById('password');
+const warnMsg = document.getElementById('warnMsg');
+const loginBtn = document.getElementById('loginBtn');
+const emptyText = '';
+
+// Messages
+const MESSAGES = {
+  empty: 'Email or password is blank. Please enter in full',
```

Comment:
Message.empty??? Could you use another name? Btw, message should be Please enter all email and password.